life_expectancy/tests/test_cleaning.py

```diff
@@ -0,0 +1,23 @@
+"""Tests for the cleaning module"""
+import os
+import sys
+
+import pandas as pd
+from life_expectancy.cleaning import clean_data, load_data, save_data
+from . import OUTPUT_DIR
+
+current_dir = os.path.dirname(os.path.abspath(__file__))
+source_dir = os.path.abspath(os.path.join(current_dir, ".."))
+sys.path.append(source_dir)
+
+def test_clean_data(pt_life_expectancy_expected):
+    """Run the `clean_data` function and compare the output to the expected output"""
+    data = load_data()
```

Comment:
my understanding is that the assignmet asks us to create a clan_data function that encapsulates the logic
````
data = load_data()
data = clean_data(['pt'], data)
save_data(data)
```

Reply:
Regarding this point, seeing the readme from assignment 2 my understanding was that there should be three functions (load, clean, save) and all are called sequentially from a main function.